### DISCUSSION TITLE
Need help to setup the project on my laptop

### DISCUSSION THREAD

**Discussion by SujalRaunak** [JdeRobot/RoboticsAcademy#2691] [Q&A] [answered]
I am facing issues. Also on running the script, there is an src folder that contains the same code again. So where should I install dependencies and run the project locally. Do I need to setup docker.
![image](https://github.com/user-attachments/assets/2ffa6b1c-6348-4cc4-9f3d-aa0626790da1)

**Comment by dduro2020** [comment, marked as answer]
It looks like you made a mistake following the installation steps. 

```bash
sh scripts/develop_academy.sh -r <link to the RAM repo/fork> -b <branch of the RAM repo> -i <humble/noetic>
```

When we use RA we refer to Robotics Academy, but if we use RAM, we are talking about Robotics Application Manager, it seems you got them confused and probably copied the link to the RA repository.

For your first tests, you can launch the script without arguments, it will prepare the src folder with the humble-devel branch of RAM.
